apps/zotonic_core/src/models/m_sysconfig.erl

```diff
@@ -31,8 +31,11 @@
 
 %% @doc Fetch the value for the key from a model source
 -spec m_get( list(), z:context() ) -> {term(), list()}.
-m_get([ Key | Rest ], _Context) ->
-    {z_config:get(Key), Rest};
+m_get([ Key | Rest ], Context) ->
+    case z_acl:is_admin(Context) of
+        true -> {z_config:get(Key), Rest};
+        false -> {undefined, Rest}
+    end;
```

Comment:
Wow, this module already existed? :wink:

Reply:
Yes, surprise, surprise :-)